### PR TITLE
✨ spotlight sort & order

### PR DESCRIPTION
**Pull request by roiLeo**
Will it blend?

### CODE REVIEW

File: src/resolvers/spotlight.ts [hunk 1]
```diff
@@ -12,22 +27,23 @@ export class SpotlightResolver {
   async spotlightTable(
     @Arg('limit', { nullable: true }) limit: number,
     @Arg('offset', { nullable: true }) offset: string,
```
Comment (roiLeo) [inline src/resolvers/spotlight.ts:29]:
```suggestion
    @Arg('limit', { nullable: true, defaultValue: null }) limit: number,
    @Arg('offset', { nullable: true, defaultValue: null }) offset: string,
```

Comment (roiLeo) [reply src/resolvers/spotlight.ts]: image

**Comment by kkukelka** [comment]
@roiLeo is this ready? 👁️

**Comment by roiLeo** [comment]
> @roiLeo is this ready? 👁️

yep it should be

**Pull request merged**